apps/judicial-system/web/src/components/PageHeader/PageHeader.tsx

```diff
@@ -11,6 +11,7 @@ const PageHeader: React.FC<Props> = (props) => {
   return (
     <Head>
       <title>{title}</title>
+      <meta name="viewport" content="initial-scale=1.0, width=device-width" />
```

Comment:
Is this needed both here and in _document? I'd think this is only needed in _document 🤔